deployment/packer/driver.py

```diff
@@ -0,0 +1,100 @@
+"""Helper functions to handle AMI creation with packer"""
+
+import boto3
+from boto import config
+import os
+from os.path import dirname
+import subprocess
+import re
+import logging
+
+LOGGER = logging.getLogger(__file__)
+LOGGER.addHandler(logging.StreamHandler())
+LOGGER.setLevel(logging.INFO)
+
+
+def get_recent_batch_ami(aws_profile):
+    """Finds the most recent Deep Learning AMI"""
+    filters = [
+        {'Name': 'architecture', 'Values': ['x86_64']},
+        {'Name': 'owner-id', 'Values': ['898082745236']},
+        {'Name': 'virtualization-type', 'Values': ['hvm']},
+    ]
+    sess = boto3.Session(profile_name=aws_profile)
+    ec2 = sess.client("ec2")
+    images = filter(
+        lambda x: re.search("Deep Learning AMI Amazon Linux .*", x['Name']),
```

Comment:
Unfortunately, the naming convention for the Deep Learning AMI includes the version and month of release (i.e. `Deep Learning AMI Amazon Linux 2.3_Jun2017`), so we can't hardcode a `Name` value into the filters.